src/components/Comment/Comment.tsx

```diff
@@ -89,6 +88,7 @@ export const avatar = (avatarSize: number): string => css`
 `;
 
 export const Comment = ({ comment, pillar }: Props) => {
+  const commentControlsButtonStyles = commentControlsButton(pillar);
```

Comment:
I'd prefer removing this abstraction and just inlining the function call. It will make it easier to trace the code with one less step to follow

Reply:
this is a minor optimisation, but defining it once here we avoid having 3 different css classes that do the same thing.

Reply:
Ah, I see what you mean. I suspect though that each call to the function will return the same string - emotion (as far as I remember without trawling the interwebs) hashes the css to get the classname so if the css is the same each time, so is the classname.

If not, then we probably want to have a conversation about the best way to make this optimisation and if we think the extra cognitive load of the abstraction is worth it - there's lots of places in the codebase where we could do this but currently don't